src/controllers/DataSetView.js

```diff
@@ -23,7 +23,13 @@ export default class DataSetView extends ViewController {
         morph: morph,
         rows: datarows,
         fields: (context.object.fields ? context.object.fields : []),
-        ...context
+        pagination: {pageLength: 50},

```

Comment:
What is this working around, or is this just a sane default value? I see it's also (?) in the constructor, should it be config or opts?